### PR TITLE
lru and cache save features post

### PR DESCRIPTION
**Pull request by czoido**

### CODE REVIEW

**Review by RubenRBS — APPROVED** [review]

File: _posts/2023-11-28-Conan-new-features-2-0-14.markdown [hunk 1]
```diff
@@ -0,0 +1,133 @@
+---
+layout: post
+comments: false
+title: "New Features in Conan Useful in CI and Production Environments"
```
Comment (memsharded) [inline _posts/2023-11-28-Conan-new-features-2-0-14.markdown:4]: Title reads a bit weird grammatically, I'd say ``New Conan features for CI and ...`` or something like that

**Review by memsharded — APPROVED** [review]
Very good

File: _posts/2023-11-28-Conan-new-features-2-0-14.markdown [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+can pass a reference pattern (like in the [conan
+list](https://docs.conan.io/2/reference/commands/list.html) command) or a [package-list
+json
+file](https://docs.conan.io/2/examples/commands/pkglists.html#examples-commands-pkglists).
+Let's save the latest revision of all packages in the Conan cache:
+
+
+```bash
+# save the last revision of all packages in the cache
+$ conan cache save "*/*:*"
```
Comment (czoido) [inline _posts/2023-11-28-Conan-new-features-2-0-14.markdown:53]:
```suggestion
$ conan cache save "*/*:*" --file=conan_cache_save_linux.tgz
```

**Review by franramirez688 — APPROVED** [review]
This reads great.

**Pull request merged**